### ISSUE TITLE
Disappearance of some buttons (Save, Download, etc.)

### ISSUE THREAD

**Issue by iG8R** [NC22/KellyC-Return-YouTube-Dislikes#28]
The title speaks for itself.
https://www.youtube.com/watch?v=KehJFetyXHs
![image](https://github.com/user-attachments/assets/0d012118-056c-442b-b4f9-916c334cd41b)

**Comment by jpa102** [comment]
youtube has some code somewhere where if there's too many items, it gets hidden in the three dots button

see this comment in issue 1074 in anarios's return youtube dislike extension: https://github.com/Anarios/return-youtube-dislike/issues/1074#issuecomment-2297484639 why it behaves like that

**Comment by Paprikar** [comment]
> youtube has some code somewhere where if there's too many items, it gets hidden in the three dots button
> 
> see this comment in issue 1074 in anarios's return youtube dislike extension: [Anarios/return-youtube-dislike#1074 (comment)](https://github.com/Anarios/return-youtube-dislike/issues/1074#issuecomment-2297484639) why it behaves like that

Theres a mile of space on the page, doesnt really look like an original behavior.

**Comment by jpa102** [comment]
try resizing the browser window while youtube is running without the extension, you'll see that it gets hidden

**Comment by Paprikar** [comment]
I just wanna point out that without extension buttons arent getting hidden.
And theres plenty of space in both cases.

**Comment by jpa102** [comment]
> behaves like its getting an infinite witdh.

yeah, that's why it's causing a "bad loop" in return youtube dislike extension from anarios that they had to make the "extra" buttons hidden

**Comment by jpa102** [comment]
> behaves like its getting an infinite witdh.

yeah, that's why it's causing a "bad loop" in return youtube dislike extension from anarios that they had to make the "extra" buttons hidden

> And theres plenty of space in both cases.

let me show you something (using this extension)

![image](https://github.com/user-attachments/assets/a624e746-9329-44e3-8724-256b3ffa1ee1)


as you can see in the highlighted element, you'll see that it has a line that makes it "half"

from what i can understand, some youtube code checks if there's more "space" to add a extra action button (in the right area, [source comment](https://github.com/Anarios/return-youtube-dislike/issues/1074#issuecomment-2308562537))

the line looks like that because in the html, the "owner" (channel) and the "actions" (actions toolbar) are put together using `display: flex`

also a screenshot with anarios's extension enabled:

![image](https://github.com/user-attachments/assets/c81674a2-36b0-4889-994c-1741985c11bc)

screenshots with zoom percentage set to 50% (this extension and anarios's extension)

![image](https://github.com/user-attachments/assets/0c1faaf5-ae7f-413d-8c01-7eb1624a9380)

![image](https://github.com/user-attachments/assets/2db818ec-9b3b-4135-91b2-7ccb4aa3734c)

my brain is a little bit fuzzy writing this comment, so this may be wrong

**Comment by Paprikar** [comment]
> my brain is a little bit fuzzy writing this comment, so this may be wrong

TY, now it makes more sense...

**Comment by jpa102** [comment]
that's okay, i've seen people being confused about [revanced](https://www.github.com/ReVanced/revanced-patches) and [revanced extended](https://www.github.com/inotia00/revanced-patches) on reddit (they're mostly talking about youtube)

they're also confused what is vanced and revanced too

and by the way, anarios's return youtube dislike - userscript version doesn't hide the extra buttons unlike the extension version (i have no idea why, i even made a fork of it)

**Comment by jpa102** [comment]
that's okay, i've seen people being confused about [revanced](https://www.github.com/ReVanced/revanced-patches) and [revanced extended](https://www.github.com/inotia00/revanced-patches) on reddit (they're mostly talking about youtube)

they're also confused what is [vanced](https://vancedapp.com) and [revanced](https://revanced.app) too

and by the way, anarios's return youtube dislike - userscript version doesn't hide the extra buttons unlike the extension version (i have no idea why, i even made a fork of it)
